new.html

```diff
@@ -13,12 +13,14 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
     <!-- POPULATES PROJECTDATA SECTION -->
     <script id="projectData-template" type="text/x-handlebars-template">
-      <article data-category="{{category}}">
-        <h1>{{{title}}}</h1>
+      <article class="clearfix" data-author="{{author}}" data-category="{{category}}">
+        <header>
+        <h1><address><a href="{{titleUrl}}">{{title}}</a></address></h1>
         <div class="byline">
           <address><a href="{{repoUrl}}">{{repoUrl}}</a></address>
-          <p>By: Susan Harrison {{publishStatus}}.</p>
-        </div>
+          <section class="author">By: {{author}} {{publishStatus}}</section>
+      </div>
+      </header>
         <section class="project_body">{{{body}}}</section>
```

Comment:
change to kebab case for unified naming conventions. project_body to project-body. Remember to update associated CSS and JS calls.